post_condor.yml

```diff
@@ -36,17 +36,17 @@ targets:
   1_get_data_ready/out/merged_flow.rds.ind:
     command: gd_get(target_name)
     
-  1_get_condor_data/out/master_list.rds:
-    command: saveRDS(object= I(readRDS(as_data_file("1_get_data_ready/out/merged_flow.rds.ind"))), file = I("1_get_condor_data/out/master_list.rds"))
+  master_list_orig:
+    command: get_master_list("1_get_data_ready/out/merged_flow.rds.ind")
+
+  1_get_condor_data/out/master_list.rds.ind:
+    command: save_master_list(master_list_orig, target_name)
 
   1_get_condor_data/out/modeled.rds.ind:
-    command: get_data("1_get_condor_data/out/master_list.rds", target_name)
+    command: get_data(master_list_orig, "1_get_condor_data/out/master_list.rds.ind", target_name)
```

Comment:
can you talk me through your reasoning in these 6 lines? I see that 1_get_data_ready/out/merged_flow.rds.ind and 1_get_condor_data/out/master_list.rds.ind are indicators for exactly the same contents (though stored in different places). What made that duplication necessary?

Reply:
I wanted to separate the pre/post modeling "master list". 

So, first I get the master list from before we sent the data to be modeled on condor. Then after we run the models, I wanted to start working on the post-processing of successfully modeled sets. I could save over in the "1_data_ready/out" folder, but I thought it might make more sense to duplicate it to keep all the modeled info in the "1_get_condor_data" folder, rather than have to go back to the "get_data_ready" folder. But...if that is more confusing than less...I could save it back in the original folder (not going to lie...I was also worried saving over the original table might have un-intended consequences, but I guess this is the project to figure out that kind of stuff).

Reply:
changing the contents of the original table might indeed have undesired consequences - remake would try to rebuild the empty version if you ever ran the first remake file again. i guess i was just confused by the fact that the two tables happen to have truly identical contents right now (same hash) - sounds like there could be times when the contents differed between the two, right? in which case I'm now fully on board with your current approach.

Reply:
Hmm....yeah, they shouldn't be totally identical...there should be a new column in the "post-condor" one....let me check what's going on...